xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -130,7 +130,7 @@
 #define ADDON_INSTANCE_VERSION_PERIPHERAL_DEPENDS     "addon-instance/Peripheral.h" \
                                                       "addon-instance/PeripheralUtils.h"
 
-#define ADDON_INSTANCE_VERSION_PVR                    "8.0.1"
+#define ADDON_INSTANCE_VERSION_PVR                    "8.0.2"
```

Comment:
Not sure ADDON_INSTANCE_VERSION_PVR_MIN must also be bumped, as a new enum value was inserted not at the end of the enum.